src/cfml/system/services/ServerService.cfc

```diff
@@ -1037,14 +1037,14 @@ component accessors="true" singleton {
 
 		serverInfo.trayOptions.prepend( { 'label':'Open...', 'items': openItems, "image" : expandPath('/commandbox/system/config/server-icons/open.png' ) } );
 
-		// serverInfo.trayOptions.prepend( { 'label' : 'Restart Server', 'hotkey':'R', 'action' : 'restartserver', 'image': expandPath('/commandbox/system/config/server-icons/home.png' ) } );
+		serverInfo.trayOptions.prepend( { 'label' : 'Restart Server', 'hotkey':'R', 'action' : "runAsync" , "command" : "box server restart " & "'#serverInfo.name#'", 'image': expandPath('/commandbox/system/config/server-icons/home.png' ), 'workingDirectory': defaultwebroot} );
```

Comment:
@mgmathus Let's see if we can find a proper "restart" icon for this.  The "home" one doesn't make sense.  Perhaps something with the two little arrows in a circle?